common/script/ops/openMysteryItem.js

```diff
@@ -28,17 +27,13 @@ module.exports = function openMysteryItem (user, req = {}, analytics) {
     });
   }
 
-  if (typeof window !== 'undefined') {
-    if (!user._tmp) user._tmp = {};
-    user._tmp.drop = item;
-  }
-
   if (req.v2 === true) {
     return user.items.gear.owned;
   } else {
     return [
       user.items.gear.owned,
       i18n.t('mysteryItemOpened', req.language),
+      item,
```

Comment:
@SabreCat You manually tested mystery items? I'm with @paglias, I think the item is just being ignored and not actually being passed on. 

I say we call it a bug and fix the data being passed back. Better to do it now than down the line. Let's check with @vIiRuS and make sure the mobile apps are not using the mystery box route.